### PR TITLE
prototype changes

### PR DESCRIPTION
**Pull request by Esshahn**
Changes to the Landing Page About Section:

-- removes repo GdK 
-- removes repo Berlin Open Source
++ adds repo Stadtpuls
++ adds repo icon for Stadtpuls
-- removes github const user (wasn't used)
== changes the text in the prototype section (not 100% final as it needs confirmation from Ben)
== changes the header image
== renames "unit counter" to "Zähler"

### CODE REVIEW

File: src/components/LandingLabAbout/index.tsx [hunk 1]
```diff
@@ -17,19 +16,13 @@ const AboutTitle: FC = ({ children }) => (
 
 const furtherProjects = [
```
Comment (dnsos) [inline src/components/LandingLabAbout/index.tsx:17]: This variable name doesn't make sense anymore. Let's change it to somethink like `repositories`?

File: src/components/LandingLabAbout/index.tsx [hunk 2]
```diff
@@ -75,39 +72,6 @@ export const LandingLabAbout: FC = () => {
...
-                    ].join(" ")}
-                  >
-                    <GitHub />
-                  </strong>
-                </span>
-              </a>
-            </div>
-
             <div className='lg:col-span-2 grid lg:grid-cols-2 gap-4 xl:gap-8'>
               {furtherProjects.map(({ name, repo, logo }) => (
```
Comment (dnsos) [inline src/components/LandingLabAbout/index.tsx:76]: The variable name needs to be updated here as well.

**Review by dnsos — CHANGES_REQUESTED** [review]
Great updates, thanks @Esshahn!

There is one small naming issue that I commented on. To keep the code understandable in the future, we should change this now.

File: src/components/LandingLabAbout/index.tsx (hunk 1 above)
Comment (Esshahn) [reply src/components/LandingLabAbout/index.tsx]: I just love these quality reviews. Thank you @dnsos !

**Review by dnsos — APPROVED** [review]

**Pull request merged**